db_stress_tool/db_stress_test_base.cc

```diff
@@ -2428,6 +2458,29 @@ void StressTest::TestCompactFiles(ThreadState* thread,
   }
 }
 
+void StressTest::TestPromoteL0(ThreadState* thread,
+                               ColumnFamilyHandle* column_family) {
+  int target_level = thread->rand.Next() % options_.num_levels;
+  Status s = db_->PromoteL0(column_family, target_level);
```

Comment:
Never heard of this API. Hope it works.

Reply:
There is still internal use case though not big. I found this API useful to move files trivially to non-L0 in UT https://github.com/facebook/rocksdb/pull/12628/files#diff-68a832436b821ea3d361ae2ab06d28f663569e352a4ca37cadbc30a0f7bd23c3R7630 ...